firmware/lib/ArduinoSerialDevice/src/ArduinoSerialDevice.cxx

```diff
@@ -11,6 +11,17 @@ bool ArduinoSerialDevice::ser_connect(uint32_t baud_rate)
     return true;
 }
 
+void ArduinoSerialDevice::ser_flush()
+{
+    // Wait for 10 ms for last bits of data
+    delay(10);
```

Comment:
I realize this is a very short delay, but do we foresee any issue this might cause?

Reply:
It's less than ideal, but this function is generally only called once at setup if at all, so it shouldn't cause any problems and I don't have an alternative.